src/pydvl/influence/frameworks/twice_differentiable.py

```diff
@@ -1,11 +1,28 @@
 from abc import ABC
-from typing import Callable, Generic, List, Sequence, Tuple, TypeVar
+from dataclasses import dataclass
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Generic,
+    List,
+    NamedTuple,
+    Sequence,
+    Tuple,
+    TypeVar,
+)
 
 TensorType = TypeVar("TensorType", bound=Sequence)
 ModelType = TypeVar("ModelType")
 DeviceType = TypeVar("DeviceType")
 
 
+@dataclass
+class iHVPResult(Generic[TensorType]):
```

Comment:
What is the reason to use a dataclass, instead of a tuple x, info ? In the end it is a question of taste, as we are breaking the function signature anyway:)